qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/create/CreateCustomerOutput.groovy

```diff
@@ -1,26 +1,14 @@
 package life.qbic.portal.portlet.customers.create
 
+import life.qbic.portal.portlet.UseCaseOutput
+
 /**
  * Output interface for the {@link life.qbic.portal.portlet.customers.create.CreateCustomer} use
  * case
  *
  * @since: 1.0.0
  * @author: Tobias Koch
  */
-interface CreateCustomerOutput {
-
-    /**
-     * Sends success notifications that have been
-     * recorded during the customer creation use case.
-     * @param notification containing a success message
-     */
-    void successNotification(String notification)
-
-    /**
-     * Sends failure notifications that have been
-     * recorded during the customer creation use case.
-     * @param notification containing a failure message
-     */
-    void failNotification(String notification)
```

Comment:
why did you remove them?

Reply:
I moved them to the parent class but can undo that. Just thought that these methods like should be there for every use case output?

Reply:
Ah, just found them! 👍

Reply:
f73c06a